code/modules/surgery/surgery.dm

```diff
@@ -196,7 +196,7 @@
 		if(AStar(E.loc, M.loc, /turf/proc/Distance, 2, simulated_only = 0))
 			germs++
 
-	if(tool.blood_DNA && tool.blood_DNA.len) //germs from blood-stained tools
+	if(!isnull(tool) && tool.blood_DNA && tool.blood_DNA.len) //germs from blood-stained tools
```

Comment:
a simple `if(tool && ...)` works here

Reply:
Yeah, I know but I think it would be better to do it explicitly

Reply:
Standard for checking if something is not null and is of the type you want it to be is to use `if(thingy)`. Using `isnull(thingy)` is handy to check if a value is null and not 0 for instance. Yet in this case it doesn't matter much and both need to be caught.
isnull is also a proc call which is more expensive compared the the old and faithful `if(tool)` check.